client/src/components/Query.jsx

```diff
@@ -497,37 +536,107 @@ export default function Query() {
 
   // Set example queries in SearchBar
   const getExample = (i) => {
-    switch (i) {
-      case 1:
-        setQuery({
-          mode: "path",
-          species: "txid7227",
-          protein: "egfr",
-          goTerm: "Wnt signaling pathway",
-          k: "4",
-        });
-        setActiveModeButton("path");
-        break;
-      case 2:
-        setQuery({
-          mode: "node",
-          species: "txid7227",
-          protein: "flw",
-          goTerm: "apical constriction",
-          k: "7",
-        });
-        setActiveModeButton("node");
-        break;
-      case 3:
-        setQuery({
-          mode: "path",
-          species: "txid7227",
-          protein: "flw",
-          goTerm: "myosin II binding",
-          k: "3",
-        });
-        setActiveModeButton("path");
-        break;
+    if (query.species == "txid7227") {
+      switch (i) {
+        case 1:
+          setQuery({
+            mode: "path",
+            species: "txid7227",
+            protein: "egfr",
+            goTerm: "Wnt signaling pathway",
+            k: "4",
+          });
+          setActiveModeButton("path");
+          break;
+        case 2:
+          setQuery({
+            mode: "node",
+            species: "txid7227",
+            protein: "flw",
+            goTerm: "apical constriction",
+            k: "7",
+          });
+          setActiveModeButton("node");
+          break;
+        case 3:
+          setQuery({
+            mode: "path",
+            species: "txid7227",
+            protein: "flw",
+            goTerm: "myosin II binding",
+            k: "3",
+          });
+          setActiveModeButton("path");
+          break;
+      }
+    }
+    else if (query.species == "txid7955") {
+      switch (i) {
+        case 1:
+          setQuery({
+            mode: "node",
+            species: "txid7955",
+            protein: "Mothers against decapentaplegic homolog 2 ",
+            goTerm: "DNA damage response",
+            k: "4",
+          });
+          setActiveModeButton("node");
+          break;
+        case 2:
+          setQuery({
+            mode: "node",
+            species: "txid7955",
+            protein: "Mothers against decapentaplegic homolog 1 ",
+            goTerm: "positive regulation of cell development",
+            k: "4",
+          });
+          setActiveModeButton("node");
+          break;
+        case 3:
+          setQuery({
+            mode: "node",
+            species: "txid7955",
+            protein: "Mothers against decapentaplegic homolog 5 ",
+            goTerm: "neural plate pattern specification",
+            k: "7",
+          });
```

Comment:
this case does not work. after updating your neo4j import files and setup w/ oliver's new changes, search for protein w/ name "smad5". looks like the alt.name changed. use this as an example:

http://localhost:5173/query?mode=node&species=txid7955&protein=smad5&goTerm=positive+regulation+of+cell+development&k=4